index.html

```diff
@@ -72,10 +74,10 @@
 			<thead>
 				<tr>
 					<th>Name</th>
-					<th>Diagonal</th>
+					<th>Diagonal (inches)</th>
 					<th>Resolution</th>
-					<th>DPI</th>
-					<th>dppx</th>
+					<th><span title="Dots Per Inch">DPI</span></th>
```

Comment:
Why not add the tooltip on the `th` directly? We don’t need a separate element here.